### PR TITLE
weekly image cve patching from Sec Pipelines

### PR DESCRIPTION
**Pull request by HDYA**
weekly image cve patching from Sec Pipelines

### CODE REVIEW

File: vhdbuilder/packer/kube-proxy-images.json [hunk 1]
```diff
@@ -5,16 +5,20 @@
...
         "multiArchVersions": [
+          "1.21.14-hotfix.20220620.1",
           "1.21.9-hotfix.20220420.2",
           "1.21.9-hotfix.20220601.1",
-          "1.21.14-hotfix.20220620.1",
-          "1.22.6-hotfix.20220330.3",
-          "1.22.6-hotfix.20220330.4",
           "1.22.11-hotfix.20220620.1",
-          "1.23.5-hotfix.20220331.2",
+          "1.22.4-hotfix.20220330.4",
```
Comment (andyliuliming) [inline vhdbuilder/packer/kube-proxy-images.json:12]: 1.22.4 was removed?

**Review by andyliuliming — APPROVED** [review]

File: vhdbuilder/packer/kube-proxy-images.json (hunk 1 above)
Comment (HDYA) [reply vhdbuilder/packer/kube-proxy-images.json:12]: yes, in https://github.com/Azure/AgentBaker/commit/8168e8f7d905188a48a2ef83a3cbf4289e90cc5d

Comment (ganeshkumar5699) [reply vhdbuilder/packer/kube-proxy-images.json:12]: Yes, we will be removing support for 1.22.4 (and older patch versions) in this week's RP release.

Comment (HDYA) [reply vhdbuilder/packer/kube-proxy-images.json]: updated and removed 1.22.4 and 1.23.3 from this PR, thanks for your confirmation @ganeshkumar5699

**Pull request merged**